src/Huginn.jl

```diff
@@ -12,7 +12,8 @@ using Tullio
 using Infiltrator
 using Plots, PlotThemes
 Plots.theme(:wong2) # sets overall theme for Plots
-# using CairoMakie, GeoMakie
+using Makie
```

Comment:
Does this work without specifying CairoMakie? I thought it was mandatory to choose one of the backends.